packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -378,6 +391,14 @@ const useIconStyles = makeStyles({
     height: '24px',
     width: '24px',
   },
+  selected: {
+    [`& .${iconClassNames.filled}`]: {
+      display: 'inline',
+    },
+    [`& .${iconClassNames.regular}`]: {
```

Comment:
This makes me a bit worried, but your reasoning does make sense and I suppose it'd be easy enough to take a dependency on react-icons in the future if it becomes a problem.

Reply:
Yes - always easier to add the dependency than take it away :)